modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormFieldTemplateContextFactory.java

```diff
@@ -148,6 +148,17 @@ private DDMFormFieldRenderingContext _createDDDMFormFieldRenderingContext(
 		ddmFormFieldRenderingContext.setProperties(ddmFormFieldTemplateContext);
 		ddmFormFieldRenderingContext.setProperty(
 			"changedProperties", changedProperties);
+
+		Long ddmFormInstanceRecordId = _ddmFormRenderingContext.getProperty(
```

Comment:
Could we use GetterUtil.getLong() here? I think it is safer and would decrease one condition in the IF below. What do you think?

Reply:
Since this is a minor change, you can approve this PR after answering this comment :+1:

Reply:
@marcelabc I tried but I have to _cast_ when using `GetterUtil.getLong`. 

If I don't cast, `_ddmFormRenderingContext.getProperty("ddmFormInstanceRecordId")` will be interpreted as a `String` during compile time and we will get the following error during runtime: `Cannot convert String to Long`

So I have to do:

```
GetterUtil.getLong((Long)_ddmFormRenderingContext.getProperty("ddmFormInstanceRecordId"))
```

Is it weird to use `getLong` and convert to `Long`?